bindings/rust/lib.rs

```diff
@@ -414,13 +60,21 @@ mod tests {
     fn can_load_grammar() {
         let mut parser = tree_sitter::Parser::new();
         parser
-            .set_language(&language())
+            .set_language(&LANGUAGE.into())
             .expect("Error loading Markdown language");
         let mut inline_parser = tree_sitter::Parser::new();
         inline_parser
-            .set_language(&inline_language())
+            .set_language(&INLINE_LANGUAGE.into())
             .expect("Error loading Markdown language");
     }
+}
+
+#[cfg(test)]
+#[cfg(feature = "parser")]
+mod parser_tests {
```

Comment:
Move the parser tests to the other file too.

Reply:
🤦🏻

Reply:
In f51080b376803e4adce9a5fb9c8b9f0ebbff4762.